app/controllers/admin/api_docs/services_controller.rb

```diff
@@ -93,28 +96,37 @@ def destroy
 
   private
 
-    def find_service
-      @api_docs_service = current_account.api_docs_services.find_by_id_or_system_name!(params[:id])
-    end
+  def api_docs_params(*extra_params)
+    permit_params = %i[name body description published skip_swagger_validations service_id] + extra_params
+    params.require(:api_docs_service).permit(*permit_params)
+  end
 
-    def swagger_spec
-      if @api_docs_service.specification.swagger_2_0?
-        @api_docs_service.specification
-      else
-        resource_listing
-      end
-    end
+  def find_service_apis_collection
+    @service_apis = current_account.services
```

Comment:
I'd say `services.accessible` https://github.com/3scale/porta/blob/master/app/models/service.rb#L91

Reply:
You are right 👍